models/example/fct_order_items.sql

```diff
@@ -0,0 +1,38 @@
+{# 
+create a fct_order_items table joining orders to order items. 
+select all fields from order items and join in the order_date, 
+email and customer_id from the orders table
+#}
+with orderitems as (
+
+    select * from {{ ref('order_items_upload') }}
```

Comment:
this should be ref the staging models that you built! `stg_orders` and `stg_order_items`